main.scss

```diff
@@ -58,7 +58,4 @@ $o-expander-is-silent: true !default;
 	.o-expander__toggle {
 		@include oExpanderToggle;
 	}
-
-	// Don't output twice
-	$o-expander-is-silent: true !global;
```

Comment:
Why was this removed?

Reply:
@JakeChampion My stupidity. 

I noticed it was imported twice in `demos/src/demo.scss` and couldn't figure out why. I assumed it was a mistake and this added to prevent output twice in the demo. 

But I see now why this is helpful for other clients, and that it is included in the  component boilerplate. Will revert. Thanks for catching!